TypeError: Cannot set property 'writingInProgress' of undefined

Running the latest develop branch as of today results in this fatal error when using a MySQL database:

```
[2015-05-20 08:45:44.210] [ERROR] console - TypeError: Cannot set property 'writingInProgress' of undefined
at /srv/etherpad-lite/src/node_modules/ueberDB/CacheAndBufferLayer.js:488:56
at /srv/etherpad-lite/src/node_modules/ueberDB/node_modules/async/lib/async.js:199:13
at /srv/etherpad-lite/src/node_modules/ueberDB/node_modules/async/lib/async.js:105:25
at /srv/etherpad-lite/src/node_modules/ueberDB/node_modules/async/lib/async.js:196:17
at Query._callback (/srv/etherpad-lite/src/node_modules/ueberDB/node_modules/async/lib/async.js:467:34)
at Query.Sequence.end (/srv/etherpad-lite/src/node_modules/ueberDB/node_modules/mysql/lib/protocol/sequences/Sequence.js:96:24)
at Query._handleFinalResultPacket (/srv/etherpad-lite/src/node_modules/ueberDB/node_modules/mysql/lib/protocol/sequences/Query.js:144:8)
at Query.OkPacket (/srv/etherpad-lite/src/node_modules/ueberDB/node_modules/mysql/lib/protocol/sequences/Query.js:78:10)
at Protocol._parsePacket (/srv/etherpad-lite/src/node_modules/ueberDB/node_modules/mysql/lib/protocol/Protocol.js:271:23)
at Parser.write (/srv/etherpad-lite/src/node_modules/ueberDB/node_modules/mysql/lib/protocol/Parser.js:77:12)
```

Rolling back to an older commit revision allows etherpad-lite to run normally. Sorry, did not track down which commit causes it to fail.

--- a/src/node/utils/toolbar.js
+++ b/src/node/utils/toolbar.js
@@ -7,7 +7,16 @@ var _ = require("underscore")
   , Button
   , ButtonsGroup
   , Separator
-  , defaultButtonAttributes;
+  , defaultButtonAttributes
+  , removeItem;
+
+removeItem = function(array,what) {
+  var ax;
+  while ((ax = array.indexOf(what)) !== -1) {
+    array.splice(ax, 1);
+  }
+ return array;
+};
 
 defaultButtonAttributes = function (name, overrides) {
   return {
@@ -244,7 +253,7 @@ module.exports = {
         buttons = []
       }else{
         // Remove Save Revision from the right menu
-        buttons[0].removeItem("savedrevision");
+        removeItem(buttons[0],"savedrevision");
       }
     }
 
@@ -253,15 +262,4 @@ module.exports = {
     });
     return groups.join(this.separator());
   }
-};
-
-Array.prototype.removeItem = function() {
-  var what, a = arguments, L = a.length, ax;
-  while (L && this.length) {
-    what = a[--L];
-    while ((ax = this.indexOf(what)) !== -1) {
-      this.splice(ax, 1);
-    }
-  }
-  return this;
 };

